### PR TITLE
feat(events-v2): Update results per-page to 50

### PR DESCRIPTION
**Pull request by lynnagara**
Align results per page on events with releases and other resources.
Ref: SEN-721

### CODE REVIEW

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export function getQuery(view, location) {
   data.field = [...new Set(fields)];
   data.groupby = groupby;
   data.sort = view.data.sort;
-
+  data.per_page = 50;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx]: Do we explicitly set per_page to 50 on frontend for other resources?

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx]: We do, and we seem to use 50 most commonly (with the notable exception of the issue list)

Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx]: Would this be better as an application level constant?

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx]: I think so, will add it here

**Review by markstory — APPROVED** [review]

**Pull request merged**